<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IngresarFact</class>
 <widget class="QDialog" name="IngresarFact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anexar a Factura</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l1_boton_cargarFactura">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Factura</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>651</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Cantidad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="l1_line_fechaVencimiento_2"/>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="l1_linea_cantidad_2"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="l1_linea_nombre_2"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Codigo</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="l1_linea_costo_2"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="l1_linea_codigo_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Costo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="l1_linea_laborat_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Fecha de Venc.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="l1_linea_principioAct_2"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Principio</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Laboratorio</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="l1_boton_anexar_2">
      <property name="text">
       <string>Anexar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingreso de Factura</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l1_boton_atras">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>701</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>80</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Proveedor</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="l1_line_proveedor"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nro.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="l1_line_fechaFactura"/>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="l1_linea_nroFact"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
